player/src/main/java/com/tritondigital/player/Provisioning.java

```diff
@@ -65,10 +66,12 @@ abstract class Server
         }
     }
 
-    private static final String DOMAIN_NAME_PROD = "playerservices.streamtheworld.com";
+    private static final String DOMAIN_NAME_PROD = "exampleProdserver.com";
```

Comment:
This is the wrong prod domain name. The playerservices.streamtheworld.com was correct.